lib/pos_blockchain_test.go

```diff
@@ -1471,7 +1472,139 @@ func _verifyCommitRuleHelper(testMeta *TestMeta, committedBlocks []*BlockHash, u
 	}
 }
 
-func _generateBlockAndAddToBestChain(testMeta *TestMeta, blockHeight uint64, view uint64, seed int64) *MsgDeSoBlock {
+func TestProcessBlockPoS(t *testing.T) {
+	testMeta := NewTestPoSBlockchainWithValidators(t)
+
+	// Create a bad block and try to process it.
+	dummyBlock := _generateDummyBlock(testMeta, 12, 12, 887)
+	success, isOrphan, missingBlockHashes, err := testMeta.chain.processBlockPoS(dummyBlock, 12, true)
+	require.False(t, success)
+	require.False(t, isOrphan)
+	require.Len(t, missingBlockHashes, 0)
+	require.Error(t, err)
```

Comment:
Just to make the order of blocks a bit easier to eyeball, it's helpful to section off all work for each block into separate code block (squiggly brackets). Really helps readability
```suggestion
	{
		// Create a bad block and try to process it.
		dummyBlock := _generateDummyBlock(testMeta, 12, 12, 887)
		success, isOrphan, missingBlockHashes, err := testMeta.chain.processBlockPoS(dummyBlock, 12, true)
		require.False(t, success)
		require.False(t, isOrphan)
		require.Len(t, missingBlockHashes, 0)
		require.Error(t, err)
	}
```